store.go

```diff
@@ -33,13 +34,15 @@ var (
 type RepoStore struct {
 	repositories map[string]*AsyncRepoCloner
 	mutex        sync.RWMutex
+	repoPath     string
```

Comment:
`repoPath` is a bit ambiguous. Maybe something like `fsPath` or `storeLocation`?

Reply:
Perhaps `repoDir` hints that it is a directory on the filesystem? WDYT @DanielMorsing?

Reply:
`repoDir` works for me :+1: